### ISSUE TITLE
Document what `sassc` is and how to get it

### ISSUE THREAD

**Issue by fasterthanlime** [itchio/itch#106]
from #104 : 

We're using [sassc](https://github.com/sass/sassc), the official command line driver (not the ruby gem). It's incredibly fast, we're happy with it!

Here's how I usually install it:

```
cd /somewhere/cool
git clone https://github.com/sass/libsass.git
cd libsass
git clone https://github.com/sass/sassc
CC=gcc make -j4
CC=gcc sudo make install
```

It probably installs in /usr/local by default? can't remember. Make sure it's in your $PATH.

**Comment by matiasbeckerle** [comment]
I was thinking of adding it too hehe :)

**Comment by matiasbeckerle** [comment]
BTW are you using Grunt at some point @fasterthanlime? I have been able to run the application without it.

**Comment by fasterthanlime** [comment]
> BTW are you using Grunt at some point? I have been able to run the application without it.

yes, for packaging: https://github.com/itchio/ci.itch.ovh/blob/master/src/jobs/itch.yml

**Comment by matiasbeckerle** [comment]
Got it. Thanks.

**Comment by fasterthanlime** [comment]
Fixed in 87ab49375ef9f6f1b06a8dc98a21f57ffa49192f
